gapic/src/main/com/google/gapid/widgets/ImagePanel.java

```diff
@@ -1279,7 +1279,7 @@ public void accept(double value) {
 
         for (int bin = 0; bin < numBins; bin++) {
           float value = data.histogram.get(channel, bin);
-          value = (float)Math.pow(value, 0.3); // Exaggerate small values.
```

Comment:
unrelated change

Reply:
Thanks, I thought I had removed that change, but obviously not.

Reply:
Fixed